recipe/meta.yaml

```diff
@@ -30,7 +29,8 @@ test:
   imports:
```

Comment:
When removing noarch, I think wheel should be added as a host dependency.
In the run section, the pinning of python should be removed. Instead `skip: True # [py<3.7]` should be set